docs/index.md

```diff
@@ -1,4 +1,164 @@
+For example to get the jsonb associated with a cell line
+```postgresql
+SELECT body FROM files WHERE dcp_schema_type='cell_line';
+``` 
+or
+```postgresql
+SELECT body FROM cell_line;
+```
+To simplify queries, the view tables for each schema type contain only the most recent version of each metadata file as the schema of the metadata may change between versions 
+
+Possible schema_types include 
+`cell_line`,
+`cell_suspension`,
+`differentiation_protocol`,
+`dissociation_protocol`,
+`donor_organism`,
+`ipsc_induction_protocol`,
+`library_preparation_protocol`,
+`links`,
+`organoid`,
+`process`,
+`project`,
+`sequence_file`,
+`sequencing_protocol`,
+`specimen_from_organism`,
+`supplementary_file`,
+`analysis_file`,
+`analysis_process`,
+`analysis_protocol`,
+`collection_protocol`,
+`enrichment_protocol`,
+`biomaterial`,
+`file`,
+`image_file`,
+`imaged_specimen`,
+`imaging_preparation_protocol`,
+`imaging_protocol`
+
+
+# Example Queries
+Get a list of all of the tables
+```postgresql
+SELECT table_name
+FROM information_schema.tables
+WHERE table_type = 'BASE TABLE'
+ AND table_schema NOT IN ('pg_catalog', 'information_schema')
+ AND table_schema = 'public';
+```
+Get total number of bundles
+```postgresql
+SELECT count(*) from bundles;
+```
+Get all data for 10 bundles
+```postgresql
+SELECT * FROM bundles LIMIT 10;
+```
+Select the data for a particular bundle
+if you use the uuid this may return multiple versions of the bundle, to only get one use the fqid (which is the version concatenated to the uuid)
+```postgresql
+SELECT * FROM bundles WHERE uuid='000f989a-bea2-46cb-9ec1-b4de8c292931'
+```
+## Querying the metadata body
+The metadata itself is stored in the jsonb format, for help with the syntax of querying jsonb check out this [cheat sheet](https://hackernoon.com/how-to-query-jsonb-beginner-sheet-cheat-4da3aa5082a3).
+
+Get a list of submissions for a particular submitter
+```postgresql
+SELECT p.uuid, p.body->'project_core'->>'project_title' AS title
+FROM project AS p
+WHERE p.body @> '{"contributors": [{"contact_name": "Aviv,,Regev"}]}'
+   OR p.body @> '{"contributors": [{"contact_name": "Sarah,,Teichmann"}]}';
+```
+
+## Querying the experimental graph
+If you have a fastq file and you are curious about its inputs you can use a custom sql function `get_all_parents` to get all of the processes that led to its creation. For example:
+
+If your sequence file has a uuid of `0b978294-b297-4a2f-bc2a-b5b67ab1f71e` you can run 
+```postgresql
+SELECT * FROM process_file_join_table 
+WHERE file_uuid = '0b978294-b297-4a2f-bc2a-b5b67ab1f71e';
+```
+This should return a list of all the processes the file is associated with. Find a row that says `OUTPUT_ENTITY` and copy the `process_uuid`
+
+Then run the following to get a list of all of its parent processes
+```postgresql
+SELECT * FROM  get_all_parents('f7755994-8ba5-4d79-b9fb-7274f0dbfc52')
+```
+- Direct parents are processes whose output becomes the input for the current process
```

Comment:
Please introduce the list with something like "Things to note about the query above:"

Reply:
👍